Utilities/bin_patch.h

```diff
@@ -215,7 +215,7 @@ class patch_engine
 	void append_title_patches(std::string_view title_id);
 
 	// Apply patch (returns the number of entries applied)
-	std::basic_string<u32> apply(const std::string& name, std::function<u8*(u32, u32)> mem_translate, u32 filesz = -1, u32 min_addr = 0);
+	std::vector<u32> apply(const std::string& name, std::function<u8*(u32, u32)> mem_translate, u32 filesz = -1, u32 min_addr = 0);
```

Comment:
Actually because the result is used in insertation almost exclusively, better to append the result to incoming vector argument reference.